infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Workflow.java

```diff
@@ -0,0 +1,28 @@
+package org.corfudb.infrastructure.orchestrator;
+
+import java.util.List;
+
+/**
+ *
+ * A workflow is an abstract container that specifies a series of steps that achieves a
```

Comment:
Is it necessarily ordered?

Reply:
Yes, for now. We can revise the workflow model, if this doesn't suffice future use cases.

Reply:
You're defining the interface. These things are hard to change once selected.

Reply:
Its not a public interface. It can be altered as we go, its hard to get the right abstraction for all the use cases in the future.

Reply:
It's still the interface everyone is going to base all these workflows off of...